projects/core/src/occ/adapters/checkout/converters/occ-order-normalizer.ts

```diff
@@ -1,14 +1,17 @@
 import { Injectable } from '@angular/core';
-import { Occ } from '../../../occ-models/occ.models';
+import { Order, OrderEntry } from '../../../../model/order.model';
+import { ReplenishmentOrder } from '../../../../model/replenishment-order.model';
+import { PRODUCT_NORMALIZER } from '../../../../product/connectors/product/converters';
 import {
   Converter,
   ConverterService,
 } from '../../../../util/converter.service';
-import { Order, OrderEntry } from '../../../../model/order.model';
-import { PRODUCT_NORMALIZER } from '../../../../product/connectors/product/converters';
+import { Occ } from '../../../occ-models/occ.models';
 
 @Injectable({ providedIn: 'root' })
-export class OccOrderNormalizer implements Converter<Occ.Order, Order> {
+export class OccOrderNormalizer
```

Comment:
Just putting it out there. Do we want to use the same normalizer? We might, I am just asking.

Reply:
I believe it would make sense to keep it the same occ normalizer since at most they have the same properties and we have checks to verify if they exist. In the end, if we make another normalizer for it, it would end up being the same as this. 

What do you think?

Reply:
Well it doesn't have the `consignments` or `unconsignedEntries` parts line 28 and 38.

Reply:
will create new normalizer